stratum/pipelines/ptf/scapy_exec.bzl

```diff
@@ -1,18 +0,0 @@
-# Copyright 2019 NoviFlow Inc.
```

Comment:
Are we removing files under stratum/pipelines/ since latest stratum doesn't have this? 
Are we rebasing the code to latest stratum as well as part of this PR